core/src/main/java/com/linecorp/armeria/common/websocket/WebSocketWriter.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.websocket;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.common.stream.StreamMessageAndWriter;
+
+/**
+ * A {@link WebSocket} that you can write {@link WebSocketFrame}s to it.
+ * Use {@link WebSocket#streaming()} to construct.
+ */
+@UnstableApi
+public interface WebSocketWriter extends WebSocket, StreamMessageAndWriter<WebSocketFrame> {
+
+    /**
+     * Write a {@link TextWebSocketFrame} to this {@link WebSocket}.
+     *
+     * @see WebSocketFrame#ofText(String)
+     */
+    default void write(String text) {
```

Comment:
How about adding `tryWrite` variants as well?

Reply:
`void write(T o);` and `boolean tryWrite(T o);` methods are at the `StreamWriter` interface which is the super interface of this interface.
I also didn't override those methods to rewrite the documentation because the Javadoc in the `StreamWriter` explains it well. But please let me know if you think we need to override those methods. 😉